jwst/assign_wcs/util.py

```diff
@@ -521,8 +521,8 @@ def create_grism_bbox(input_model,
     ----------
     input_model : `jwst.datamodels.ImagingModel`
         Data model which holds the grism image
-    reference_files : dict
-        Dictionary of reference files
+    reference_file : str
```

Comment:
I have a slight preference of keeping this a dictionary. It's more general and in case we need another reference file in the future we won't have to change the interface again.

Reply:
Yes, half way through the process of ripping out the "genericness" that thought occurred to me too. I guess I'll put it back in and make sure all the other lower-level functions still treat it appropriately.

Reply:
No reference files at all are needed for the branch of extract_2d that handles NIRSpec (FS and MOS), because wavecorr is now in its own step. So the only functions that need any ref files included in their calls are the ones for tso and grism, which I'll restore.